test/integration/logs.test.js

```diff
@@ -0,0 +1,73 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const app = require("../../server");
+const addUser = require("../utils/addUser");
+const authService = require("../../services/authService");
+const cleanDb = require("../utils/cleanDb");
+
+const config = require("config");
+const cookieName = config.get("userToken.cookieName");
+
+const userData = require("../fixtures/user/user")();
+
+chai.use(chaiHttp);
+
+describe("Logs", function () {
+  let jwt;
+
+  afterEach(async function () {
+    await addUser();
+  });
+
+  after(async function () {
+    await cleanDb();
+  });
+
+  describe("GET /logs/member/cache/self", function () {
+    before(async function () {
+      await cleanDb();
+      const userId = await addUser(userData[0]);
+      jwt = authService.generateAuthToken({ userId });
+    });
+
+    it("Should return logs of the last 24 hours of user", function (done) {
+      chai
+        .request(app)
+        .get("/logs/member/cache/self")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((error, response) => {
+          if (error) {
+            return done(error);
+          }
+
+          expect(response).to.have.status(200);
+          expect(response.body).to.be.a("object");
+          expect(response.body.message).to.be.a("string");
+          expect(response.body.count).to.be.a("number");
+          expect(response.body.logs).to.be.an("array");
```

Comment:
Can we add some test cache logs in the `before` block and test out the array content.

Reply:
Yeah but model function job is to fetch data while here we are testing middleware, routes, controller, models and anything that goes to making that API.

Reply:
Added some test logs and also added one more test